resource/tasks.json

```
@@ -1886,6 +1886,7 @@
             ["^(?:[<4~]|《|。)+", ""],
             ["^c(?!ast)", ""],
             ["^\\^一", ""],
+            ["^_", ""],
```

```suggestion
            ["^_(?=.)", ""],
```

The possibility of needing to OCR a single `_` is very small, but just in case what about a lookahead?